### PR TITLE
Improve instructions in Contributing guide

### PR DESCRIPTION
**Pull request by ccordoba12**
- I added some instructions to work with conda envs.
- I also updated some instructions that were outdated.
- I fixed rst formatting in several places.
- Finally, I fixed some minor spelling errors.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -49,28 +49,41 @@ These are the basic steps needed to start developing on QDarkStyle.
...
-          cd qdarkstyle
+      git clone https://github.com/USERNAME/qdarkstyle
+      cd qdarkstyle
 
 -  Checkout the appropriate branch
 
    .. code:: bash
 
-          git checkout master
+      git checkout master
```
Comment (dpizetta) [inline CONTRIBUTING.rst:59]: @ccordoba12 Could you please change to develop until we have all the resources generated in the server? Until that, I need to generate them in my machine. Tks

**Review by dpizetta — COMMENTED** [review]
Thanks for the corrections and review.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (ccordoba12) [reply CONTRIBUTING.rst]: Done @dpizetta. I also moved the branch of this PR from `master` to `develop`. Is that ok? or do you prefer to keep it in `master`?

Comment (dpizetta) [reply CONTRIBUTING.rst]: It is ok, tks!

**Comment by dpizetta** [comment]
Let me know when it's ready to merge, tks

**Comment by ccordoba12** [comment]
This is ready @dpizetta

**Pull request merged**